integration-tests/src/test/java/oracle/kubernetes/operator/ItImageTool.java

```diff
@@ -0,0 +1,192 @@
+// Copyright (c) 2019, Oracle Corporation and/or its affiliates.  All rights reserved.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.kubernetes.operator;
+
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.util.Map;
+
+import oracle.kubernetes.operator.utils.Domain;
+import oracle.kubernetes.operator.utils.ExecCommand;
+import oracle.kubernetes.operator.utils.ExecResult;
+import oracle.kubernetes.operator.utils.Operator;
+import oracle.kubernetes.operator.utils.TestUtils;
+import org.junit.AfterClass;
+import org.junit.Assume;
+import org.junit.BeforeClass;
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.runners.MethodSorters;
+
+/**
+ * Simple JUnit test file used for testing Operator.
+ *
+ * <p>This test is used for creating Operator(s) and domain(s) which are managed by the Operator(s).
+ * WebLogic docker image is created by WebLogic Image Tool
+ */
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+public class ItImageTool extends BaseTest {
+  private static final String TEST_RESOURCE_LOC = "integration-tests/src/test/resources";
+  private static String TEST_APP_PROPS_FILE = "OperatorWIT.properties";
```

Comment:
Can you add new properties in existing OperatorIT.properties like weblogicImageTagWIT, weblogicImageNameWIT, weblogicImageNameWIT instead of adding new properties file as its duplicating all other properties. Currently there is only one place to change the project properties in OperatorIT.properties. By adding a new properties file, if we want to change the properties like maxIterationsPod, waitTimePod, etc for the entire project we will have to modify both the properties files.

Reply:
changed and local test passed:

[INFO] Running oracle.kubernetes.operator.ItImageTool
12-02-2019 12:11:18 INFO  oracle.kubernetes.operator.BaseTest <clinit> QUICKTEST true FULLTEST true
12-02-2019 12:11:18 INFO  oracle.kubernetes.operator.ItImageTool staticPrepare Loading props from: OperatorIT.properties
...
12-02-2019 12:21:40 INFO  oracle.kubernetes.operator.BaseTest logTestBegin BEGIN testCustomImageUsed
12-02-2019 12:21:40 INFO  oracle.kubernetes.operator.ItImageTool testCustomImageUsed Command to get pod's image name: kubectl get pod domainonpvwlst-admin-server -n default -o=jsonpath='{.spec.containers[*].image}'
12-02-2019 12:21:40 INFO  oracle.kubernetes.operator.ItImageTool testCustomImageUsed WebLogic Docker image used by pod <domainonpvwlst-admin-server> is <imagetool/build/weblogic:12.2.1.3.0>
12-02-2019 12:21:40 INFO   oracle.kubernetes.operator.ItImageTool testCustomImageUsed SUCCESS - testCustomImageUsed